packages/ccip-js/test/integration-hedera.test.ts

```diff
@@ -0,0 +1,117 @@
+import { expect, it, beforeAll, describe } from '@jest/globals'
+import * as CCIP from '../src/api'
+import * as Viem from 'viem'
+import { hederaTestnet } from 'viem/chains'
+import bridgeToken from '../artifacts-compile/BridgeToken.json'
+
+const ccipClient = CCIP.createClient()
+const bridgeTokenAbi = bridgeToken.contracts['src/contracts/BridgeToken.sol:BridgeToken'].bridgeTokenAbi
+
+const HEDERA_TESTNET_RPC_URL = process.env.HEDERA_TESTNET_RPC_URL || 'https://testnet.hashio.io/api'
+const SEPOLIA_CHAIN_SELECTOR = '16015286601757825753'
+
+describe('Integration: Hedera -> Sepolia', () => {
```

Comment:
`'Integration: Hedera -> Sepolia (viem)`? Make it explicitly viem only? and do we not want to test for ethers?  Perhaps we can test both by using different clients in different it() blocks?  Ugly but serves the same purpose as fulled duped tests without the duping?

Reply:
I'll mark it as viem for now and I want to revisit the tests between the providers in another PR.